src/data/full-quiz.ts

```diff
@@ -7152,5 +7152,27 @@ const fullQuiz = [
       "Npm scripts allow commandline tools to be called directly without relying on external plugins.",
     Link: "https://www.freecodecamp.org/news/why-i-left-gulp-and-grunt-for-npm-scripts-3d6853dd22b8/",
   },
+  {
+    Question:
+      "Which of the following type of Hackers look for vulnerable computers over the internet, exploit them and use them for their advantage?",
+    Answer: "Black Hat Hackers",
+    Distractor1: "Red Hat Hackers",
+    Distractor2: "White Hat Hackers",
+    Distractor3: "Grey Hat Hackers",
+    Explanation:
+      "Black Hat Hackers exploit vulnerable computers over the internet by installing backdoors, malwares, sending phishing emails and links , therby getting access to private and sensitive data",
+    Link: "https://www.freecodecamp.org/news/white-hat-black-hat-red-hat-hackers/",
+  },
+  {
+    Question:
+      "Which of the following type of Hackers protect civilians and organizations by finding and reporting vulnerabilites in the system?",
+    Answer: "White Hat Hackers",
+    Distractor1: "Grey Hat Hackers",
+    Distractor2: "Black Hat Hackers",
+    Distractor3: "Red Hat Hackers",
+    Explanation:
+      "White Hat Hackers work for organizations as Cybersecurity Engineers, Penetration Tester, Security Analyst, CISO etc where they perform tasks like scanning the network, configuring Intrution Detection Systems, ethically hacking the system to find and report vulnerabilites, programming honeypots and monitoring the network for any suspicious activity",
```

Comment:
I think we should cut this sentence down because it will probably look a little long when it is in the answer modal.  
If the user wants to learn more, they can click on the learn more link in the answer more which will provide more detailed explanations. 

```suggestion
      "White Hat Hackers work for organizations as Cybersecurity Engineers, Penetration Testers, etc.  where they perform tasks like ethically hacking the system to find and report vulnerabilities. ",
```